product_docs/docs/edbcloud/beta/reference/index.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Using the EDB Cloud API
+---
+
+Use the EDB Cloud API to integrate directly with EDB Cloud for management activities such as cluster provisioning, de-provisioning, and scaling.
+
+The API reference documentation is available from the [EDB Cloud portal](https://portal.edbcloud.com). The direct documentation link is [https://portal.edbcloud.com/api/docs/](https://portal.edbcloud.com/api/docs/).
+
+To access the API, you need a token. The high-level steps to obtain a token are:
+1. [Query the authentication endpoint](http://localhost:8000/edbcloud/latest/reference/#query-the-authentication-endpoint).
+2. [Request the device code](http://localhost:8000/edbcloud/latest/reference/#request-the-device-code).
+3. [Authorize as a user](http://localhost:8000/edbcloud/latest/reference/#authorize-as-a-user).
+4. [Request the token](http://localhost:8000/edbcloud/latest/reference/#request-the-token).
+
+
+
+## Query the Authentication Endpoint
+
+Use the following command to get the information returned by the authentication endpoint that you need later:
+
+```
+curl https://portal.edbcloud.com/api/v1/auth/provider
+```
+    
+The response returns the `clientId`, `issuerUri`, `scope`, and `audience`. For example:
+```
+{
+  "clientId": "pM8PRguGtW9yVnrsvrvpaPyyeS9fVvFh",
+  "issuerUri": "https://edbcloud.us.auth0.com",
+  "scope": "openid profile email offline_access",
+  "audience": "https://portal.edbcloud.com/api"
+}
+```
+
+## Request the Device code
+
+Use the following call to get a device code: 
+```
+   curl --request POST \
+     --url 'https://edbcloud.us.auth0.com/oauth/device/code' \
```

Comment:
plz change this to: `https://auth.edbcloud.com/oauth/device/code`, we are going to use customized domain name